src/DIRAC/Core/Utilities/test/Test_File.py

```diff
@@ -107,10 +108,10 @@ def testGetMD5ForFiles():
 def testGetCommonPath():
```

Comment:
This test, I guess, passes. Does it really need to be here? (we are not testing the standard library...)

Reply:
I mostly left it to make sure the standard library did what DIRAC expected rather than something subtly different. Now it's passed once I agree it can be removed.